### PR TITLE
New versions for k8s client and dependency new versions

### PR DESCRIPTION
**Pull request by danushkaf**

### CODE REVIEW

File: okhttp-ws/2.7.5.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ ~ Copyright (c) 2005-2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (manoj-kumara) [inline okhttp-ws/2.7.5.wso2v1/pom.xml]: Header should be as,
Copyright (c) 2016, WSO2 Inc. (http://wso2.com) All Rights Reserved.

File: kubernetes-client/1.3.99.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ ~ Copyright (c) 2005-2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (manoj-kumara) [inline kubernetes-client/1.3.99.wso2v1/pom.xml]: Header should be as,
Copyright (c) 2016, WSO2 Inc. (http://wso2.com) All Rights Reserved.

File: logging-interceptor/2.7.5.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ ~ Copyright (c) 2005-2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (manoj-kumara) [inline logging-interceptor/2.7.5.wso2v1/pom.xml]: Header should be as,
Copyright (c) 2016, WSO2 Inc. (http://wso2.com) All Rights Reserved.

File: okhttp/2.7.5.wso2v1/pom.xml [hunk 4]
```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ ~ Copyright (c) 2005-2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (manoj-kumara) [inline okhttp/2.7.5.wso2v1/pom.xml]: Header should be as,
Copyright (c) 2016, WSO2 Inc. (http://wso2.com) All Rights Reserved.

File: okhttp-ws/2.7.5.wso2v1/pom.xml (hunk 1 above)
Comment (danushkaf) [inline okhttp-ws/2.7.5.wso2v1/pom.xml]: Fixed

File: kubernetes-client/1.3.99.wso2v1/pom.xml (hunk 2 above)
Comment (danushkaf) [inline kubernetes-client/1.3.99.wso2v1/pom.xml]: Fixed

File: logging-interceptor/2.7.5.wso2v1/pom.xml (hunk 3 above)
Comment (danushkaf) [inline logging-interceptor/2.7.5.wso2v1/pom.xml]: Fixed

File: okhttp/2.7.5.wso2v1/pom.xml (hunk 4 above)
Comment (danushkaf) [inline okhttp/2.7.5.wso2v1/pom.xml]: Fixed

**Pull request merged**